### ISSUE TITLE
nvimr won't do anything

### ISSUE THREAD

**Issue by statquant** [jalvesaq/Nvim-R#215]
I installed nvim 2.0 on fedora 25
I installed nvim-r via neobundle
I did the same with nerdtree which works

Pressing \rf does not do anything
I run it from terminator (no tmux involved)

![screenshot from 2017-07-05 19-16-33](https://user-images.githubusercontent.com/3612473/27878543-813d4000-61b6-11e7-9131-0c67b5c0ebbf.png)

**Comment by jalvesaq** [comment]
Nvim-R' s documentation is displayed, so the installation seems to be OK, but since there is no error message, I don't know what might be wrong. Perhaps someone who uses Neobundle can help you.

**Comment by statquant** [comment]
isn't there something I can do like checking runtime paths starting in
debug, run tests?

**Comment by jalvesaq** [comment]
I will soon replace all `runtime` commands with `source`, and this might make it less likely to happen conflicts with old Nvim-R installations.

**Comment by statquant** [comment]
Ok I'll wait some time to start using Nvim-R and neovim then.
Thanks

**Comment by jalvesaq** [comment]
@statquant Could you test again, please?

**Comment by statquant** [comment]
Yes better, still a small bug somewhere though 
![screenshot from 2017-07-08 21-06-30](https://user-images.githubusercontent.com/3612473/27988731-632ad524-6421-11e7-98f5-ad0afecd7c05.png)

**Comment by jalvesaq** [comment]
Different plugin managers source the scripts in different order. That's why not all bugs appear for everyone. I guess it is fixed now.

**Comment by statquant** [comment]
Hello, sorry for my late reply.
unfortunately we seem to be back to square 1.
I do not get anything...
Is there a minor versioning somewhere to make sure I get the lattest ?
(although I am fairly sure I do)

**Comment by jalvesaq** [comment]
There is no variable storing the Nvim-R version, but I installed Neobundle and, then installed Nvim-R through Neobundle and everything works fine here. You can try these commands in Neovim (Normal mode) and look for something suspect:
```
:echo &filetype
:messages
:echo rplugin_home
:echo maplocalleader
```
You should get:
  1. The filetype. It should be `r`, `rnoweb` or `rmd` otherwise the plugin will not be enabled.
  2. An error message that has flashed very quickly to be seen at first (if there is any).
  3. The path to Nvim-R installation.
  4. Your local leader (it should be `\` otherwise the command `\rf` will not work).

**Comment by jalvesaq** [comment]
Looking at the first screenshot that you have posted, it looks like that you are calling neobundle#begin and neobundle#end twice. They should be called only once.

**Comment by statquant** [comment]
Gosh !!!! really sorry, don't know how this eluded me :(
Removing this stupidity and it works like a charm.
Sorry again

**Comment by jalvesaq** [comment]
Thanks for your feedback!
